feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/controller/confirm/ConfirmSetControllerViewModel.kt

```diff
@@ -0,0 +1,108 @@
+package jp.co.soramitsu.feature_staking_impl.presentation.staking.controller.confirm
+
+import androidx.lifecycle.liveData
+import androidx.lifecycle.viewModelScope
+import jp.co.soramitsu.common.address.AddressIconGenerator
+import jp.co.soramitsu.common.base.BaseViewModel
+import jp.co.soramitsu.common.mixin.api.Validatable
+import jp.co.soramitsu.common.resources.ResourceManager
+import jp.co.soramitsu.common.utils.inBackground
+import jp.co.soramitsu.common.validation.ValidationExecutor
+import jp.co.soramitsu.feature_account_api.presenatation.actions.ExternalAccountActions
+import jp.co.soramitsu.feature_staking_impl.R
+import jp.co.soramitsu.feature_staking_impl.domain.StakingInteractor
+import jp.co.soramitsu.feature_staking_impl.domain.staking.controller.ControllerInteractor
+import jp.co.soramitsu.feature_staking_impl.domain.validations.controller.SetControllerValidationPayload
+import jp.co.soramitsu.feature_staking_impl.domain.validations.controller.SetControllerValidationSystem
+import jp.co.soramitsu.feature_staking_impl.presentation.StakingRouter
+import jp.co.soramitsu.feature_staking_impl.presentation.common.fee.FeeStatus
+import jp.co.soramitsu.feature_staking_impl.presentation.common.mapFeeToFeeModel
+import jp.co.soramitsu.feature_staking_impl.presentation.staking.controller.set.bondSetControllerValidationFailure
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.launch
+
+class ConfirmSetControllerViewModel(
+    private val router: StakingRouter,
+    private val controllerInteractor: ControllerInteractor,
+    private val addressIconGenerator: AddressIconGenerator,
+    private val _payload: ConfirmSetControllerPayload,
```

Comment:
Why underscore prefix is used here? It seems like there is not need for it

Reply:
Done